apps/docs/src/guide/creating-a-fuel-dapp/index.md

```diff
@@ -137,17 +140,17 @@ bun run dev
 
 :::
 
-You should now be able to see the counter dApp running at `http://localhost:5173`. You can try changing the contents of the `./sway-programs/contract/src/main.sw` file and see the changes reflected in the UI without having to restart the server.
+You should now be able to see the dApp running at `http://localhost:5173`. Go ahead and connect a wallet to the dApp. You can choose the Burner Wallet from the list if you don't want to connect a wallet.
```

Comment:
Reckon a picture here would be helpful? Showing the list of wallets?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3178/commits/d7beaecdc81ca0eec86f6928916f9f133049723c